### PR TITLE
[Opmondev-163]: add structure for debian packaging

### PR DESCRIPTION
**Pull request by wisecrow**

### CODE REVIEW

File: opendata_collector_module/etc/opendata_sources_settings.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  # from_dt: '2022-12-05T00:00:00'
+  # until_dt: '2023-05-01T00:00:00
+  # opendata_api_tz_offset: '+0200'
+  # verify_ssl: False
+  #
+# TEST-SOURCE-2
+  # url: http://example2:/api/harvest
+  # limit: 5000
+  # from_dt: '2022-12-05T00:00:00'
+
```
Comment (justinask7) [inline opendata_collector_module/etc/opendata_sources_settings.yaml:27]:
```suggestion
```

File: opendata_collector_module/etc/opendata_sources_settings.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  # limit: 2000
+  # from_dt: '2022-12-05T00:00:00'
+  # until_dt: '2023-05-01T00:00:00
+  # opendata_api_tz_offset: '+0200'
+  # verify_ssl: False
+  #
+# TEST-SOURCE-2
+  # url: http://example2:/api/harvest
+  # limit: 5000
+  # from_dt: '2022-12-05T00:00:00'
```
Comment (justinask7) [inline opendata_collector_module/etc/opendata_sources_settings.yaml:26]: should this really be commented? 🤔

File: opendata_collector_module/setup.cfg [hunk 3]
```diff
@@ -0,0 +1,2 @@
+[metadata]
+version = attr: metrics_opendata_collector.__version__
```
Comment (justinask7) [inline opendata_collector_module/setup.cfg:2]:
```suggestion
version = attr: metrics_opendata_collector.__version__

```

**Review by justinask7 — APPROVED** [review]

File: opendata_collector_module/etc/opendata_sources_settings.yaml (hunk 2 above)
Comment (wisecrow) [reply opendata_collector_module/etc/opendata_sources_settings.yaml:26]: We don't want these values be used as real settings. I just a examples.

**Pull request merged**